src/Bootstrappers/UrlGeneratorBootstrapper.php

```diff
@@ -78,6 +79,10 @@ protected function useTenancyUrlGenerator(Tenant $tenant): void
             }
         }
 
+        // Inherit scheme (http/https) from the original generator
+        $originalScheme = Str::before($this->originalUrlGenerator->formatScheme(), '://');
+        $newGenerator->forceScheme($originalScheme);
```

Comment:
What does `formatScheme()` return if no scheme was forced? Is it just `http://`?

Wondering if we can detect if a scheme was forced so we do not force a scheme if a scheme was not forced on the original generator.